modules/lss-shareAlliancePost/ShareAlliancePost.js

```diff
@@ -713,10 +713,7 @@
                     .then(res => res.json())
                     .then(data => {
                         messages = messages.map(message => {
-                            data?.average_credits && (message = message.replace(
-                                /%CREDITS%/g,
-                                data.average_credits.toLocaleString()
-                            ));
+                            message = message.replace(/%CREDITS%/g, data.average_credits.toLocaleString()||'');
```

Comment:
This is will lead to an error if `data.average_credits` is not defined!
Please use
```js
data.average_credits?.toLocaleString() || ''
```
or an "old way" check instead.

Reply:
Changed